FirebaseRemoteConfig/Sources/Public/FirebaseRemoteConfig/FIRRemoteConfig.h

```diff
@@ -30,6 +30,20 @@ extern NSString *const _Nonnull FIRNamespaceGoogleMobilePlatform NS_SWIFT_NAME(
 extern NSString *const _Nonnull FIRRemoteConfigThrottledEndTimeInSecondsKey NS_SWIFT_NAME(
     RemoteConfigThrottledEndTimeInSecondsKey);
 
+/**
+ * Represents a listener that can be removed by calling its only method: remove. This is returned
+ * when calling `addOnConfigUpdateListener` and the method should be used when you no longer want to
+ * listen for new config updates. If this is the last listener it will close the Realtime stream.
```

Comment:
Listener registration returned by `addOnConfigUpdateListener`. Calling its method `remove` stops the listener from receiving config updates and unregisters itself.

If `remove` is called and no other listener registrations remain, the connection to the Realtime RC backend is closed. Subsequently calling `addOnConfigUpdateListener` will re-open the connection.